### PR TITLE
Remove unneeded feature attr from atomic integers doctests

### LINKED ISSUE #61448: Feature attribute can be removed from atomic integers doctests.
Since #56753 has already landed and the feature is stable, all the `#![feature(integer_atomics)]` at the top of the doctests, e.g. [here](https://doc.rust-lang.org/stable/core/sync/atomic/struct.AtomicU8.html#examples) can be removed.

This issue has been assigned to @lzutao via [this comment](https://github.com/rust-lang/rust/issues/61448#issuecomment-498014010).

### PR DESCRIPTION
**Pull request by tesuji**
Closes #61448

r? @Centril

### CODE REVIEW

**Comment by lzutao** [comment]
Worth a try!

@bors rollup

**Comment by sfackler** [comment]
Where else is $extra_feature used? It seems like it can be removed entirely from the macro, right?

**Comment by lzutao** [comment]
I was thinking that `#![feature(integer_atomics)]` still be used in unstable
Atomic integers (for example: https://doc.rust-lang.org/nightly/std/sync/atomic/struct.AtomicU8.html#method.fetch_min). Looks like I am wrong:
https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=6da6afeaf1fe1a4f018682886a0c39d4

**Comment by lzutao** [comment]
`AtomicU128` and `AtomicI128` are unstable and need `$extra_feature` in doctest.
Should I keep it? @sfackler

File: src/test/run-pass/atomic-alignment.rs [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #![feature(cfg_target_has_atomic)]
-#![feature(integer_atomics)]
```
Comment (lzutao) [inline src/test/run-pass/atomic-alignment.rs]: We probably want to keep this because `AtomicU128` is unstable.

**Comment by sfackler** [comment]
Oh, okay.

**Comment by lzutao** [comment]
Ping @sfackler , this PR is ready to review again.

**Comment by sfackler** [comment]
@bors r+ rollup

Thanks!

**Pull request merged**